src/common/github-flavored-markdown.js

```diff
@@ -1,19 +1,42 @@
 import memoizee from 'memoizee';
 import MarkdownIt from 'markdown-it';
+import replaceLink from 'markdown-it-replace-link';
 
-const MAX_MEMOIZED_VALUES = 1000;
-
-const gfm = new MarkdownIt();
-const render = memoizee(gfm.render.bind(gfm), { max: MAX_MEMOIZED_VALUES });
-const renderInline = memoizee(gfm.renderInline.bind(gfm), { max: MAX_MEMOIZED_VALUES });
+const getGfmForCdnRootUrl = memoizee(cdnRootUrl => {
+  let gfm;
+  if (cdnRootUrl) {
+    gfm = new MarkdownIt({
+      replaceLink: link => link.startsWith('cdn://') ? `${cdnRootUrl}/${link.substr(6)}` : link
```

Comment:
My understanding about this change was exactly viceversa 😄 
I thought the URLs inside the markdown sections are already environment aware and we need to:
- make them generic so they are valid in the importing system
- extract them from the markdown into `cdnResources` 

Basically what the PR title is saying 🤔 I feel like I'm missing something cause this line is making the URLs specific, right?
-